recipes/r-qs2/meta.yaml

```
@@ -0,0 +1,108 @@
+{% set version = '0.1.4' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-qs2
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/qs2_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/qs2/qs2_{{ version }}.tar.gz
+  sha256: c2dec8a8643db08ae6e496fd775fe723aba86742d9fb3fc93b65aa640e162f04
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+  missing_dso_whitelist:
```

Maybe just skip Windows for now, unless you really want it compiled for R 4.1 (the only Win variant supported on `staged-recipes`). Once in feedstock, the migrator will swing by and try adding Windows back for 4.3 and 4.4.

```suggestion
  skip: true  # [win]
  missing_dso_whitelist:
```